server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -240,9 +235,6 @@ public String executor() {
         };
 
         try {
-            // Will be replaced with ExtensionTokenProcessor and PrincipalIdentifierToken classes from feature/identity
```

Comment:
Lets leave the principal identifier token comment in place since this change doesn't address that issue

Reply:
Sure, no problem